python/requirements.txt

```diff
@@ -1,3 +1,47 @@
-astropy
-fastapi
-mangum
+annotated-types==0.6.0
+anyio==3.7.1
+architect-functions==1.0.0
+astropy==6.0.0
+astropy-healpix==1.0.2
+astropy-iers-data
+boto3==1.33.13
+botocore==1.33.13
+certifi==2023.11.17
+cffi==1.16.0
+charset-normalizer==3.3.2
+click==8.1.7
+cryptography==41.0.7
+dnspython==2.4.2
+ecdsa==0.18.0
+email-validator==2.1.0.post1
+fastapi==0.105.0
+idna==3.6
+jmespath==1.0.1
+mangum==0.17.0
+numpy==1.26.2
+packaging==23.2
+pipdeptree==2.13.1
+psycopg2-binary==2.9.9
```

Comment:
I doubt we use either of these packages. Where does this list of dependencies come from?

Reply:
pipdeptree one of yours from the original requirements.txt. psycopg2-binary is because stuff not part of this pull request (but coming in later one) uses a PostgreSQL database. I did not pull out all the unused requirements for this pull request, I will admit.

Reply:
Anyway... I pushed bc073f9 which updates requirements.txt to include only modules needed for this pull request.